WordPress/src/main/java/org/wordpress/android/models/ReaderTag.java

```diff
@@ -74,10 +75,17 @@ public String getCapitalizedTagName() {
         if (tagName == null) {
             return "";
         }
-        // HACK to allow iPhone, iPad, iEverything else
-        if (tagName.startsWith("iP")) {
+        // If already uppercase, assume correctly formatted
+        if (Character.isUpperCase(tagName.charAt(0))) {
             return tagName;
         }
+        // Accounts for iPhone, ePaper, etc.
+        if (tagName.length() > 1 &&
+                Character.isUpperCase(tagName.charAt(0)) &&
```

Comment:
I think these are reversed. Should be checking if the first character is lowercase and the second is uppercase.